changelog/548.breaking.rst

```diff
@@ -1 +1,2 @@
 Rename ``channel_id`` parameter to ``channel`` on :attr:`Guild.create_scheduled_event` and :meth:`GuildScheduledEvent.edit`.
+Allow ``entity_type`` parameter :attr:`Guild.create_scheduled_event` to be missing.
```

Comment:
Fairly sure this line would be better suited in a `590.feature.rst`.

Reply:
If we do move this, this pr should still continue to symlink 548.breaking.rst

Reply:
so we mean to remove the symlink and just move the changelog entry to `590.feature.rst`?

Reply:
Keep the symlink, but move this snippet to `590.feature.rst`

Reply:
https://github.com/DisnakeDev/disnake/pull/617/commits/1ee428f45e2ab263731f5d9b38e8b4f562930059